src/sentry/models/deletedorganization.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from django.db import models
+
+from sentry.db.models import sane_repr
+from sentry.models.deletedentry import DeletedEntry
+
+
+class DeletedOrganization(DeletedEntry):
+    name = models.CharField(max_length=64)
+    slug = models.SlugField(unique=True)
```

Comment:
This shouldn't be UNIQUE. Hypothetically, if I created an org `foo`, deleted it, then created another called `foo`, and deleted it, this will cause a problem. So since this is a log, we should have very few constraints like this.